package.json

```diff
@@ -1,148 +1,123 @@
   "categories": [
     "Linters"
   ],
-  "activationEvents": [
-    "onCommand:extension.spectral-lint",
-    "*"
-  ],
-  "license": "Apache-2.0",
-  "main": "./out/extension.js",
   "contributes": {
-    "commands": [
-      {
-        "command": "extension.spectral-lint",
-        "title": "Spectral: Lint"
-      }
-    ],
-    "yamlValidation": [
-      {
-        "fileMatch": ".spectral.yaml",
-        "url": "./schemas/ruleset.schema.json"
-      },
-      {
-        "fileMatch": ".spectral.yml",
-        "url": "./schemas/ruleset.schema.json"
-      },
-      {
-        "fileMatch": "spectral.yaml",
-        "url": "./schemas/ruleset.schema.json"
-      },
-      {
-        "fileMatch": "spectral.yml",
-        "url": "./schemas/ruleset.schema.json"
-      }
-    ],
-    "jsonValidation": [
-      {
-        "fileMatch": ".spectral.json",
-        "url": "./schemas/ruleset.schema.json"
-      },
-      {
-        "fileMatch": "spectral.json",
-        "url": "./schemas/ruleset.schema.json"
-      }
-    ],
     "configuration": {
-      "title": "Spectral",
       "properties": {
-        "spectral.lintOnSaveTimeout": {
-          "type": "integer",
-          "default": 2000,
-          "description": "Delay in ms before lint on save event is fired."
+        "spectral.enable": {
+          "default": true,
+          "description": "Controls whether or not Spectral is enabled.",
+          "scope": "resource",
+          "type": "boolean"
         },
-        "spectral.defaultRuleset": {
-          "type": "string",
-          "default": "spectral:oas",
-          "description": "Default ruleset URI."
+        "spectral.rulesetFile": {
+          "description": "Location of the ruleset file to use when validating. If omitted, the default is a .spectral.yml/.spectral.json in the same folder as the document being validated. Paths are relative to the workspace.",
+          "scope": "resource",
+          "type": "string"
         },
-        "spectral.clampRanges": {
-          "type": "boolean",
-          "default": true,
-          "description": "Minimize range highlighted by errors and warnings"
+        "spectral.run": {
+          "default": "onType",
+          "description": "Run the linter on save (onSave) or as you type (onType).",
+          "enum": [
+            "onSave",
+            "onType"
+          ],
+          "scope": "resource",
+          "type": "string"
+        },
+        "spectral.trace.server": {
+          "default": "off",
+          "description": "Traces the communication between VS Code and the language server.",
+          "enum": [
+            "off",
+            "messages",
+            "verbose"
+          ],
+          "scope": "window",
+          "type": "string"
+        },
+        "spectral.validateFiles": {
+          "description": "An array of file globs which should be validated by Spectral. If language identifiers are also specified, the file must match both in order to be validated.",
```

Comment:
@tillig Could we add here a reference to the package/syntax that's used for globbing?

Reply:
Fixed - added an example and noted it's minimatch format.